TTS/server/server.py

```diff
@@ -111,7 +111,10 @@ def convert_boolean(x):
     use_cuda=args.use_cuda,
 )
 
-use_multi_speaker = hasattr(synthesizer.tts_model, "num_speakers") and synthesizer.tts_model.num_speakers > 1
+use_multi_speaker = hasattr(synthesizer.tts_model, "num_speakers") and (
+    synthesizer.tts_model.num_speakers > 1 or synthesizer.tts_speakers_file is not None
```

Comment:
If speakers_file is not None then num speakers must be > 1. Isn't that the case?

Reply:
No, If you have just one speaker. In inference time we can use just speakers embedding from one speaker (for like speaker adaptation and zero-shot). So if we provide `tts_speakers_file` we need to consider this a multi-speaker model.